website/source/docs/platform/k8s/servers-outside-kubernetes.html.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: "docs"
+page_title: "Consul Servers Outside of Kubernetes - Kubernetes"
+sidebar_current: "docs-platform-k8s-run-servers-outside"
+description: |-
+  Running Consul servers outside of Kubernetes
+---
+
+# Consul Servers Outside of Kubernetes
+
+If you have a Consul cluster already running, you can configure your
+Consul clients inside Kubernetes to join this existing cluster.
+
+The below `config.yaml` file shows how to configure the Helm chart to install
+Consul clients that will join an existing cluster.
+
+The `global.enabled` value first disables all chart components by default
+so that each component is opt-in. This allows us to _only_ setup the client
+agents. We then opt-in to the client agents by setting `client.enabled` to
+`true`.
+
+Next, `client.exposeGossipPorts` can be set to true or false depending on if
+you want the clients to be exposed on the Kubernetes node IPs (`true`) or on
```

Comment:
Grammarly suggested that 😀 
```suggestion
you want the clients to be exposed on the Kubernetes node IPs (`true`) or
```

Reply:
Do we have to pay them for code reviews? 😆